### DISCUSSION TITLE
Problems with use on touch screens

### DISCUSSION THREAD

**Discussion by tuzbtla** [labwc/labwc#2351] [Q&A] [answered]
When I use the touch screen, I can click normally, but my mouse icon stays in one place. Is there any function similar to the touch screen mode, does not display my mouse, and there is a click feedback, how should I open it?

**Comment by jp7677** [comment, marked as answer]
This should be addressed with https://github.com/labwc/labwc/pull/2273 which isn't yet part of a release.
